detection-rules/recon_recipients_bccd_undisclosed.yml

```diff
@@ -14,6 +14,12 @@ source: |
   )
   and (
     length(subject.subject) <= 10
+    // extract the "real subject" (strip off any external warning keywords)
+    or any(regex.iextract(subject.subject,
+                          '^\[?(?:EXT|EXTERNAL)\]? ?(?P<real_subject>.*)'
```

Comment:
I like this idea, do you want to use the same regex that includes flexiblity of generic tags? 
`\[[^\]]+\]\s?){0,3}`

i think we use this in a lot of rules for tags in subjects.